src/styles/globals.css

```diff
@@ -9,9 +10,55 @@
     @apply rounded bg-gray-200 py-0.5 px-1 text-sm dark:bg-gray-800;
   }
 
+  .prose code.hljs {
+    @apply bg-transparent;
+  }
+
+  .prose a code {
+    @apply dark:text-sky-400 !important;
+  }
+
+  .prose pre {
+    @apply rounded-lg border border-transparent dark:border-gray-900 dark:bg-black;
+  }
+
+  @media (min-width: 1024px) {
+    .docs.prose pre {
+      max-width: calc(100vw - 520px);
+    }
+    .blog.prose pre {
+      max-width: calc(100vw - 352px);
+    }
+  }
+
+  @media (min-width: 1280px) {
+    .blog.prose pre {
+      max-width: calc(100vw - 672px);
+    }
+  }
+
+  @media (min-width: 1440px) {
+    .docs.prose pre {
+      max-width: calc(100vw - 840px);
+    }
+  }
+
+  @media (min-width: 1536px) {
+    .docs.prose pre {
+      max-width: 707px;
+    }
+    .blog.prose pre {
+      max-width: 832px;
+    }
+  }
```

Comment:
This CSS should fix the x-overflow issues that you caught on the "build an app" page.

[Before](https://docs-site-git-jschr-featget-started-content-raydiant.vercel.app/docs/get-started/build-an-app)
[After](https://developer-docs-git-sebko-featfix-code-syntax-raydiant.vercel.app/docs/get-started/build-an-app)